pkg/asset/installconfig/ovirt/client.go

```diff
@@ -69,3 +69,30 @@ func FetchVNICProfileByClusterNetwork(con *ovirtsdk.Connection, clusterID string
 	}
 	return nil, fmt.Errorf("there are no vNic profiles for the given cluster ID %s and network name %s", clusterID, networkName)
 }
+
+// GetClusterName returns the name of the ovirt cluster with the specified Cluster ID
+func GetClusterName(con *ovirtsdk.Connection, clusterID string) (string, error) {
+	clusterResponse, err := con.SystemService().ClustersService().ClusterService(clusterID).Get().Send()
+	if err != nil {
+		return "", err
```

Comment:
Please add context to the error here, e.g. `fmt.Errorf("failed to fetch cluster %s (%w)", clusterID, err)`

Reply:
Done